app/views/devise/registrations/new.html.erb

```diff
@@ -4,6 +4,11 @@
   <%= devise_error_messages! %>
 
   <div class="field">
+      <%= f.label :username %><br />
+      <%= f.text_field :username, autofocus: true %>
+    </div>
+
+  <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true %>
```

Comment:
Look into what autofocus does. You don't want this to be true in two places. I would suggest keeping it at the top input box